### PR TITLE
Revamp the readme to show examples for common scenarios

### PR DESCRIPTION
**Pull request by carolynvs**
Continuing https://github.com/Azure/service-catalog-cli/pull/50 after the great fork snafu of 2017.
___

For now I'm directing people to svc-cat -h instead of trying to manually duplicate the help text on the readme.

### CODE REVIEW

**Comment by carolynvs** [comment]
@arschles Okay I've addressed your comments from my pre-fork PR, plus the CI is working. 🎉

File: README.md [hunk 1]
```diff
@@ -44,58 +44,121 @@ You will need to find a permanent location for it and add it to your PATH.
...
+  azure-eventhub             Azure Event Hub Service          7bade660-32f1-4fd7-b9e6-d416d975170b
+  azure-cosmos-mongo-db      MongoDB on Azure provided by     8797a079-5346-4e84-8018-b7d5ea5c0e3a
+                             CosmosDB
+  azure-mysqldb              Azure Database for MySQL         997b8372-8dac-40ac-ae65-758b4a5075a5
+                             Service
+  azure-postgresqldb         Azure Database for PostgreSQL    b43b4bba-5741-4d98-a10b-17dc5cee0175
+                             Service
+  azuresearch                Azure Search                     c54902aa-3027-4c5c-8e96-5b3d3b452f7f
+  azure-keyvault             Azure Key Vault                  d90c881e-c9bb-4e07-a87b-fcfe87e03276
+  azure-sqldb                Azure SQL Database Service       fb9bc99e-0aa9-11e6-8a8a-000d3a002ed5
```
Comment (arschles) [inline README.md]: can you make a note here to indicate that this output would come up if you had OSBA installed

File: README.md [hunk 2]
```diff
@@ -44,58 +44,121 @@ You will need to find a permanent location for it and add it to your PATH.
...
+
+Plans:
+     NAME             DESCRIPTION
++-------------+-------------------------+
+  standard800   Standard Tier, 800 DTUs
+  basic100      Basic Tier, 100 DTUs
+  basic50       Basic Tier, 50 DTUs.
+  standard200   Standard Tier, 200 DTUs
+  standard400   Standard Tier, 400 DTUs
+  standard100   Standard Tier, 100 DTUs
```
Comment (arschles) [inline README.md]: same here RE the note about OSBA

Comment (carolynvs) [reply README.md]: I added the note to the top of the examples because it applies to every one. What do you think?

**Comment by carolynvs** [comment]
@arschles Can you take another look? I've added a comment at the top of the examples letting people know that the example output assumes that OSBA is installed since it applied to every example command.

File: README.md (hunk 2 above)
Comment (arschles) [reply README.md]: That SGTM

**Pull request merged**